performance_test/src/communication_abstractions/fast_rtps_communicator.hpp

```diff
@@ -89,8 +89,9 @@ class FastRTPSQOSAdapter
     if (m_qos.history_kind == QOSAbstraction::HistoryKind::KEEP_LAST) {
       return static_cast<int32_t>(m_qos.history_depth);
     } else if (m_qos.history_kind == QOSAbstraction::HistoryKind::KEEP_ALL) {
-      // Keep all, keeps all. No depth required, but setting to dummy value.
-      return 1;
+      // Keep all, keeps all. No depth required, but returning it in order to use it
```

Comment:
Is this comment needed? All the other branches use the same pattern and neither of them have a comment.

Reply:
Reverted this part. See answer to the other comment.